source/_components/sensor.command_line.markdown

```diff
@@ -144,3 +144,17 @@ sensor:
     command: "python3 /path/to/script/arest-value.py"
     unit_of_measurement: "°C"
 ```
+
+### {% linkable_title Usage of templating in command %}
+
+command: supports inclusion of templates. Which means you can use various states from other sensors in your commands.
```

Comment:
Not quite sure what you are referring to here, however, I have revised the text and changed small parts in my new commit